### ISSUE TITLE
[Accessibility] Charts: mutationobserver and chart region announcement

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#2338]
1. (Magma) [Keyboard focus is lost after closing the “More options” menu with the Esc key](https://jira.cengage.com/browse/A11Y-577) <- similar MutationObserver solution as for table
2. (Magma) [Chart region is not announced by screen readers and is not recognized using image navigation shortcuts](https://jira.cengage.com/browse/A11Y-609) <- can definitely do this, already partly fixed
1. (Magma) [Plot points are not keyboard operable or announced by screen readers](https://jira.cengage.com/browse/A11Y-613)
2. (MT) [Screen reader does not announce content updates triggered by checkbox interaction](https://jira.cengage.com/browse/A11Y-4109)

**Comment by olehnoskov-cengage** [comment]
[[Accessibility][JAWS][Activity Performance Report]: Screen reader does not announce content updates triggered by checkbox interaction](https://jira.cengage.com/browse/A11Y-4109) ✅ 

[[Accessibility] [Global] [Charts]: Keyboard focus is lost after closing the “More options” menu with the Esc key.](https://jira.cengage.com/browse/A11Y-577) fixed by [this PR](https://github.com/cengage/react-magma/pull/2382) ✅

**Comment by artemtrusov-cengage** [comment]
PRs [2576](https://github.com/cengage/react-magma/pull/2576) and [2586](https://github.com/cengage/react-magma/pull/2586) are related to ticket [2328](https://github.com/cengage/react-magma/issues/2328).

**Comment by serhiiyanhol-cengage** [comment]
[fix(Charts): Add focus support for inner chart content](https://github.com/cengage/react-magma/pull/2464) wasn't merged to dev.

also points should remain visible while hovered:
https://github.com/user-attachments/assets/ad918ab0-f6d4-4b40-97b6-9e481eab330d
